classes/log/td.php

```diff
@@ -90,9 +90,9 @@ public static function write($level, $msg, $method = null)
 	   	$message['call']  = $call;
 
 		$res = $logger->post('td.'.$database.'.fuel_log',$message);
-        if(!$res){
-            return false;
-        }
+		if(!$res){
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Expected 1 space after IF keyword; 0 found
* Expected 1 space after closing parenthesis; found 0